src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -3324,6 +3324,8 @@ gregtech.machine.world_accelerator.mode_entity=Entity Mode
 # General machinery
 gregtech.machine.basic.input_from_output_side.allow=Allow Input from Output Side
 gregtech.machine.basic.input_from_output_side.disallow=Disallow Input from Output Side
+gregtech.machine.muffle.on=Muffling: on
```

Comment:
I would change these translations to 'Sound Muffling: Enabled' and 'Sound Muffling: Disabled', it is more clear to the player what hitting the machine with a hammer actually does.